client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -281,9 +283,11 @@ export default function getThankYouPageUrl( {
 	}
 
 	const signupFlowName = getSignupCompleteFlowName();
+	const isDomainOnly =
+		siteSlug === 'no-site' && getAllCartItems( cart ).every( isDomainRegistration );
```

Comment:
Wondering if the check should take into account `isDomainMapping` as well? as noted in https://github.com/Automattic/wp-calypso/pull/65630/files#r922840280

Reply:
This is moot since the reference logic has been removed.